examples/grpc-envoy/src/main/resources/envoy/envoy.yaml

```diff
@@ -48,5 +48,5 @@ static_resources:
               - endpoint:
                   address:
                     socket_address:
-                      address: host.testcontainers.internal
+                      address: host.docker.internal
```

Comment:
`host.docker.internal` doesn't work on Linux. I think we should revert this change.
See https://github.com/envoyproxy/java-control-plane/pull/303

Reply:
I was merging because of a crash, but I forgot the file. I fixed it!